.golangci.yml

```diff
@@ -121,6 +121,7 @@ linters-settings:
         files:
           - '**/api/**'
           - '**/e/**'
+          - '**/lib/srv/**'
```

Comment:
nit: Revert all changes to the file, except for this line? It's shifting blame for no particular reason.